x-pack/filebeat/input/gcppubsub/input.go

```diff
@@ -109,7 +115,7 @@ func NewInput(
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Do any of the args contain the configured input name? If so it would be nice to log a deprecation warning for the old name.

Reply:
Good point! I added a `harvester.ForwarderConfig` in order to capture the configured input name. Please let me know if this looks ok to you 🙂